### ISSUE TITLE
Site Login for Chaturbate for Kodi on Firestick

### ISSUE THREAD

**Issue by marsrover1-ops** [dobbelina/repository.dobbelina#1401]
### Prerequisites

- [X] I have searched the issue tracker to avoid opening a duplicate issue.

### Description

In NC, for Chaturbate, you have to verify your age in order to view the site. Otherwise, I have to use VPN thru out of country connection. This interferes with access to some of the other Cummination sites. One of the verification options for Chaturbate is with an existing login account. 
Please add site login option to Chaturbate for Kodi on Firestick.  If there already is way to log into Chaturbate, please explain how to do that.

**Comment by 12asdfg12** [comment]
If you have a verified account, try to log in using the "sessionid" cookie.
There is also a user/password login option (click on "Followed Cams"), but this doesn't always work because of Cloudflare, and it certainly won't work if the cookie option doesn't work.

**Comment by marsrover1-ops** [comment]
Yeah, I saw the other post and tried the sessionid, but that did not work.
